phoenix-core/src/main/java/org/apache/phoenix/expression/function/RowTimestampEpochFunction.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.phoenix.expression.function;
+
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable;
+import org.apache.phoenix.expression.Expression;
+import org.apache.phoenix.parse.FunctionParseNode.BuiltInFunction;
+import org.apache.phoenix.schema.tuple.Tuple;
+import org.apache.phoenix.schema.types.PDataType;
+import org.apache.phoenix.schema.types.PDate;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Function to return the timestamp of the empty column which functions as the row timestamp. The
+ * result returned can be used for debugging(eg. using HBase shell), logging etc.
+ */
+@BuiltInFunction(name = RowTimestampEpochFunction.NAME, args = {})
+public class RowTimestampEpochFunction extends ScalarFunction {
+
+    public static final String NAME = "ROW_TIMESTAMP_EPOCH";
```

Comment:
"Epoch" is generally used as beginning/origin of a time-period. Is there another name we can use for this one? .. e.g. ROW_TIMESTAMP_STRING